### PR TITLE
type feat : Added Feedback folder

### PR DESCRIPTION
**Pull request by cypher-naman**
Feedback folder is added with register.html which will serve as the base for further development of this project. It is a new file containing the basic UI of the project for the registration page.

### CODE REVIEW

File: Feedback/templates/Feedback/register.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{%extends 'base.html'%}
```
Comment (GeekyShacklebolt) [inline Feedback/templates/Feedback/register.html]: Not a big issue. But I prefer to give space after "{% " and before " %}" in Django tags.

**Comment by cypher-naman** [comment]
delete this too.

**Comment by cypher-naman** [comment]
Is anyone seeing this? At least tell my why it is still retained.

**Comment by gutsytechster** [comment]
Hey @cypher-naman, I think since we don't require register page anymore. This feature is obsolete now. So, we should close the PR.

**Pull request closed without merging**